recipes/libassert/all/conanfile.py

```diff
@@ -0,0 +1,117 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.microsoft import check_min_vs, is_msvc_static_runtime, is_msvc
+from conan.tools.files import get, copy, rm, rmdir
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, export_conandata_patches
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+
+class PackageConan(ConanFile):
+    name = "libassert"
+    description = "The most over-engineered and overpowered C++ assertion library."
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/jeremy-rifkin/libassert"
```

Comment:
Can you add a [`package_type`](https://docs.conan.io/2/reference/conanfile/attributes.html#package-type) here please?